incubator/mariadb-cluster/README.md

```diff
@@ -66,13 +67,41 @@ The following tables lists the configurable parameters of the MariaDB chart and
 | `master.persistence.storageClass` | Persistent Volume Storage Class                     | ``                                                                |
 | `master.persistence.accessModes`  | Persistent Volume Access Modes                      | `[ReadWriteOnce]`                                                 |
 | `master.persistence.size`         | Persistent Volume Size                              | `8Gi`                                                             |
+| `master.config`                   | Config file for the MariaDB Master server           | `_default values in the values.yaml file_`                        |
 | `master.resources`                | CPU/Memory resource requests/limits for master node | `{}`                                                              |
+| `master.livenessProbe.enabled`                    | Turn on and off liveness probe (master)                                                              | `true`                               |               |
```

Comment:
It seems the table is not well aligned. Depends on your editor but the are many modules available (for instance https://atom.io/packages/markdown-table-formatter for ATOM)